### DISCUSSION TITLE
More shapes or third party shapes.

### DISCUSSION THREAD

**Discussion by GeoffPierson** [jgraph/drawio#3277] [Ideas]
I discovered this project through my employer (we're considering adopting it to replace Smartdraw).
But I also have interest in personal use.

To the latter, I have a question:
Are there any third-party object packages that can be downloaded?

The reason is for my own (personal) use: I created a home wiring diagram a while ago using my own licensed copy of Visio using their "Electrical and Telecom" object group.  And while I was able to quickly create a group in DrawIO by copying the individual objects from Visio to DrawIO and adding them to a custom group of shapes, this only saves the objects as images - not genuine shapes that can be resized cleanly, accept connections (cleanly), etc.

If there are no third parties providing 'shapes' ... are there plans within your development team to create more shapes?

Thank you.

**Comment by evanharmon1** [comment]
You can import a variety of 3rd party shape libraries:
- https://github.com/jgraph/drawio-libs
- https://www.drawio.com/blog/public-custom-libraries
